packages/npm-dependencies-validation/test/utils/fileUtil.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from "chai";
+import { toJsonObject } from "../../src/utils/fileUtil";
+
+describe("fileUtil unit tests", () => {
+  context("toJsonObject()", () => {
+    it("will return json object for valid json string", () => {
+      const content = `{"name": "test"}`;
+      type TestJson = { name: string };
+      const result: TestJson = toJsonObject<TestJson>(content);
+      expect(result).to.haveOwnProperty("name");
```

Comment:
not really important and also subjective.
But in such cases you can use `to.deep.equal({name: "test"})` and avoid the 
need for:
1. helper type definition
2. multiple assertions

Reply:
Done